### ISSUE TITLE
Conflict with Hatunnel vpn

### ISSUE THREAD

**Issue by FrancoisCam** [srilakshmikanthanp/clipbirdroid#7]
When I try to connect to Ha Tunnel, Clipbird automatically disconnects from the device where it was synced. I then have to log in manually to be able to use it again. The same thing happened when the phone disconnected from Hatunnel. I therefore propose the addition of an automatic reconnection function each time Clipbird disconnects from its group (that include on device startup, on application startup, and after any unintentional disconnection). Thanks in advance.

**Comment by srilakshmikanthanp** [comment]
Thank you for raising this issue. It appears that Clipbird is not working well with VPNs like Ha Tunnel. I will work on adding an automatic reconnection feature to ensure it reconnects on device startup, application startup, and after any unintentional disconnection.

We appreciate your patience as we resolve this.

**Comment by srilakshmikanthanp** [comment]
Can you able to provide steps to reproduce this issue?

**Comment by FrancoisCam** [comment]
Of course. It happens exactly as I said: 
- I shared my mobile connection with my PC via hotspot;
- I synchronized the clipbird application on Android with the one on my PC;
- I opened the Ha tunnel application on Android;
- I pressed the start button in Ha tunnel (to connect to VPN).
Right after the VPN Ha tunnel is connected, Clipbird disconnects automatically. It remains open but is disconnected. To send the clipboard again, it is necessary to open the Clipbird application and reconnect with the one on PC.
I wanted to send a screenshot or a recording video but Ha tunnel does not allow it.

**Comment by srilakshmikanthanp** [comment]
Thank you for the information. As a recent graduate, I have some personal commitments, including job-related tasks, but rest assured, I will prioritize fixing this as soon as possible.

**Comment by srilakshmikanthanp** [comment]
@FrancoisCam When I try to start HA Tunnel Plus, it says it failed! Is there a configuration I need to set up before connecting? Alternatively, could you try different VPN apps like Super VPN? I usually use that and haven't had any problems.

**Comment by FrancoisCam** [comment]
Yes, Ha Tunnel Plus needs a configuration to work. There are files containing predefined configurations and we import them to Ha Tunnel. The problem is that these configuration files are already expired and won't work anymore. But I have the same problem with Proton VPN. Proton VPN is free and it don't need a file configuration. Then you can work with this VPN.

**Comment by srilakshmikanthanp** [comment]
If you have ProtonVPN Plus, could you try using split tunneling and excluding the `clipbird` app to see if it works?

**Comment by FrancoisCam** [comment]
Spit tunneling is only for paid version. I can't try that at the moment. In other hand, it's better if you could fix this problem for the free version.
